### ISSUE TITLE
Context Menu using incorrect language

### ISSUE THREAD

**Issue by thorkia** [microsoft/terminal#10788]
### Windows Terminal version (or Windows build number)

1.9.1942.0

### Steps to reproduce

Have 2 keyboard languages installed - I have English and Spanish (Mexican) with the default set to English

Right Click on context menu in tab bar where no tabs are present

### Expected Behavior

Context menu displays in the correct language.  For my use case - English.

I should see: Restore/Move/Size/Minimize/Maximize/Close

### Actual Behavior

Context menu displays in the second language. For my use case - Spanish

I see: Restaurar/Mover/Tamaño/Minimizar/Maximizar/Cerrar

![incorrectLanguage](https://user-images.githubusercontent.com/9490280/126911144-1cab9841-288e-4cf5-ba99-0e7927670614.png)
![incorrectLanguage2](https://user-images.githubusercontent.com/9490280/126911146-3891e589-8804-4388-9cef-8cabf4b22a55.png)

**Comment by thorkia** [comment]
Issue also occurs in Preview version 1.10.1933.0

**Comment by vefatica** [comment]
Could it be using Spanish (Mexico) because English (Canada) is not available?

**Comment by thorkia** [comment]
@vefatica English is available.  Every other app and every other context menu uses English.  It's only that one context menu in Windows Terminal that uses it.

**Comment by vefatica** [comment]
I wondered because the "Language pack installed" and "basic typing installed" icons do not appear next to English (Canadian).

**Comment by thorkia** [comment]
@vefatica But US English is installed and the default.  But your suggestion did prompt me to install the language pack for English Canada and that resolved the issue.

It's interesting that only that one context in menu in that one app had an issue when a partially installed language pack is present.  I uninstalled Canada completely, and it keeps using English.

So there is a weird bug with how it recognizes which language to use for that one context menu.

**Comment by vefatica** [comment]
I'm glad the issue is resolved.  You're right.  It sounds a little strange.

**Comment by zadjii-msft** [comment]
Huh. Weird. Glad you got this sorted out!

**Comment by ChillFire22** [comment]
Know this is closed but posting in case someone else stumbles upon the same issue.
I got wt yesterday (on windows 10) and the context menu when I right clicked on the desktop was using another language (arabic)  which should not have happened as everything in english.

I tried removing my other languages, leaving only English (I use US English). That fixed the issue and the tw context menu appeared as expected in english. Re-added my other languages and the problem did not re-occur. (Need to see if it will though after a restart, but I don't think it should)

Hope this helps anyone.
